src/wp-admin/includes/dashboard.php

```diff
@@ -1990,70 +1990,61 @@ function wp_dashboard_empty() {}
-		<?php endif; ?>
-			<li><?php printf( '<a href="%s" class="welcome-icon welcome-learn-more">' . __( 'Learn more about getting started' ) . '</a>', __( 'https://wordpress.org/support/article/first-steps-with-wordpress/' ) ); ?></li>
-		</ul>
+	<div class="welcome-panel-header">
+		<h2><?php _e( 'Welcome to WordPress!' ); ?></h2>
+		<p>
+			<a href="<?php echo esc_url( admin_url( 'about.php' ) ); ?>">
+			<?php
+				/* translators: %s: Current WordPress version. */
+				printf( __( 'Learn more about the %s version.' ), $display_version );
+			?>
+			</a>
+		</p>
 	</div>
+	<div class="welcome-panel-column-container">
+		<div class="welcome-panel-column">
+			<div class="welcome-panel-icon-pages"></div>
+			<div class="welcome-panel-column-content">
+				<h3><?php _e( 'Author rich content with blocks and patterns' ); ?></h3>
+				<p><?php _e( 'Block patterns are pre-configured block layouts. Use them to get inspired or create new pages in a flash.' ); ?></p>
+				<a href="<?php echo esc_url( admin_url( 'post-new.php?post_type=page' ) ); ?>"><?php _e( 'Add a new page' ); ?></a>
+			</div>
+		</div>
+		<div class="welcome-panel-column">
+			<div class="welcome-panel-icon-layout"></div>
+			<div class="welcome-panel-column-content">
+			<?php if ( $is_block_theme ) : ?>
+				<h3><?php _e( 'Customize your entire site with block themes' ); ?></h3>
+				<p><?php _e( 'Design everything on your site — from the header down to the footer, all using blocks and patterns.' ); ?></p>
+				<a href="<?php echo esc_url( admin_url( 'site-editor.php' ) ); ?>"><?php _e( 'Open site editor' ); ?></a>
+			<?php else : ?>
+				<h3><?php echo ( 'Start Customizing' ); ?></h3>
+				<p><?php echo ( 'Configure your site&#8217;s logo, header, menus, and more in the Customizer.' ); ?></p>
+				<?php if ( $can_customize ) : ?>
+					<a class="load-customize hide-if-no-customize" href="<?php echo wp_customize_url(); ?>"><?php _e( 'Open the Customizer' ); ?></a>
+				<?php endif; ?>
+			<?php endif; ?>
+			</div>
+		</div>
+		<div class="welcome-panel-column">
+			<div class="welcome-panel-icon-styles"></div>
+			<div class="welcome-panel-column-content">
+			<?php if ( $is_block_theme ) : ?>
+				<h3><?php _e( 'Switch up your site&#8217;s look & feel with Styles' ); ?></h3>
+				<p><?php _e( 'Tweak your site, or give it a whole new look! Get creative — how about a new color palette or font?' ); ?></p>
+				<a href="<?php echo esc_url( admin_url( 'site-editor.php?styles=open' ) ); ?>"><?php _e( 'Edit styles' ); ?></a>
+			<?php else : ?>
+				<h3><?php echo ( 'Discover a new way to build your site.' ); ?></h3>
+				<p><?php echo ( 'There&#8217;s a new kind of WordPress theme, called a block theme, that lets you build the site you&#8217;ve always wanted &#8212; with blocks and styles.' ); ?></p>
+				<?php echo ( '<a href="https://wordpress.org/support/article/block-themes/">Learn about block themes</a>' ); ?>
```

Comment:
>Note: that last link goes to a draft support article, a user support doc for block themes, and I'm just assuming the URL. I'm pretty sure it should be in the string for translation so it can be updated to the locale site if available, but I can remove it if not.

Yes, the link should be translatable. But like you said, it's a draft support article. What's been done in the past when the article doesn't yet exist? Is it added later? Or ____?

Reply:
[Confirmed with @annezazu that the link is correct](https://github.com/WordPress/Documentation-Issue-Tracker/issues/63#issuecomment-1005008936). 🎉